### PR TITLE
Unified Syscalls

### PR DESCRIPTION
**Pull request by rennergade**
This is the ultimate merge of the past several months work of getting rid of the alternate "lind_syscalls" pathways in libc and nacl.

Let's all give it a final review before we merge it in.

### CODE REVIEW

**Review by jesings — APPROVED** [review]
This is mostly boiler-plate aside from the 2 lines that should be fixed, looks merge-ready

File: elf/Versions [hunk 1]
```diff
@@ -153,53 +154,24 @@ ld {
...
-    lind_connect;
-    lind_listen;
-    lind_sendto;
-    lind_accept;
-    lind_getpeername;
-    lind_setsockopt;
-    lind_getsockopt;
-    lind_shutdown;
-    lind_select;
     lind_getifaddrs;
```
[the remark this answers is not in the document]
Comment (kapkic) [reply elf/Versions:174]: I was thinking the same, if they are not going to be implemented let's remove it.

File: sysdeps/nacl/nacl_syscalls.h [hunk 2]
```diff
@@ -116,6 +132,33 @@
 #define NACL_sys_wait4                  122
 #define NACL_sys_sigprocmask            123
 #define NACL_sys_lstat                  124
+#define NACL_sys_gethostname            125
+#define NACL_sys_pread                  126
+#define NACL_sys_pwrite                 127
+
```
Comment (kapkic) [inline sysdeps/nacl/nacl_syscalls.h:138]: We can remove the empty lines between consecutive call numbers.

**Review by youngfalcon — APPROVED** [review]

**Comment by rennergade** [comment]
Ok this seems good to go once nacl is merged.

**Pull request merged**